spectractor/simulation/atmosphere.py

```diff
@@ -538,19 +569,22 @@ def load_file(self, filename):
                                                                NB_OZ_POINTS,
                                                                len(self.lambdas))).T, bounds_error=False, fill_value=0)
 
-    def simulate(self, ozone, pwv, aerosols):
+    def simulate(self, ozone, pwv, aerosols, angstrom_exponent=None):
         """Interpolate from the atmospheric grid to get the atmospheric transmission.
 
         First ozone, second pwv, last aerosols, to respect order of loops when generating the grid
 
         Parameters
         ----------
         ozone: float
-            Ozone quantity in Dobson
+            Ozone quantity in Dobson.
         pwv: float
-            Precipitable Water Vapor quantity in mm
+            Precipitable Water Vapor quantity in mm.
         aerosols: float
-            VAOD Vertical Aerosols Optical Depth
+            VAOD Vertical Aerosols Optical Depth.
+        angstrom_exponent: float, optional
+            Angstrom exponent for aerosols. If negative or None, default aerosol model from Libradtran is used. If value is 0.0192,
+            atmospheric transmission is very close to the case angstrom_exponent negative (default: None).
```

Comment:
I'm not quite sure what the sentence `is very close to the case angstrom_exponent negative` is supposed to mean.

Reply:
Bad english maybe... I mean that if you set 0.0192 you find quasi exactly the same result than having set None or a negative number (which deactivates the angstroù_exponent).

Reply:
Ah, OK, I see. I'd probably say `If value is 0.0192, the atmospheric transmission is very close to the case with angstrom_exponent=None` in that case, or something like that. I just didn't understand the use of `negative` there.

Reply:
Ok, I commit a change with that suggestion.